backend/src/configs.clj

```diff
@@ -0,0 +1,27 @@
+(ns configs)
+
+(def out-of-turn "Out of turn play")
+
+(def no-opp "Waiting for an opponent")
+
+(def play "Playing")
```

Comment:
Please name this `playing`, play is an action verb 😱

Reply:
It is as intended since it's the message that says the player she needs to play.